src/main/kotlin/com/sparetimedevs/consonance/config/MongodbConfiguration.kt

```diff
@@ -1,72 +1,47 @@
 package com.sparetimedevs.consonance.config
 
-import java.io.File
-import java.io.FileInputStream
-import java.util.*
-
-private const val USER_DIR = "user.dir"
-private const val USER_HOME = "user.home"
-
 private const val DB_USERNAME = "mongodb.config.username"
 private const val DB_PASSWORD = "mongodb.config.password"
 private const val DB_HOST = "mongodb.config.host"
 private const val DB_PORT = "mongodb.config.port"
 private const val DB_AUTH_SOURCE = "mongodb.config.authsource"
-private const val DB_NAME = "mongo.config.dbname"
+private const val DB_NAME = "mongodb.config.dbname"
+private const val AUTH_SOURCE = "authSource"
+private const val MONGO_DB = "mongodb"
 
-object MongodbConfiguration {
 
-    private val properties = Properties()
-    private val pathFromUserDir = "${File.separator}src${File.separator}main${File.separator}resources${File.separator}config.properties"
-    private val pathFromUserHome = "${File.separator}.con${File.separator}config${File.separator}config.properties"
-    private val configFileFromProject = System.getProperty(USER_DIR) + pathFromUserDir
-    private val configFileFromUserHome = System.getProperty(USER_HOME) + pathFromUserHome
+object MongodbConfiguration {
 
     fun load(): String {
-        readConfigFromProject()
-        //TODO should make this also load values from readConfigFromUserHome() and override values when present in the user home config.
-        return getMongodbConnectionString()
+        return getDbConnectionUrl()
```

Comment:
I don't think this `load` function is needed anymore.

Reply:
Removed.